include/Zycore/Defines.h

```diff
@@ -97,10 +97,13 @@
 #elif defined(__FreeBSD__)
 #   define ZYAN_FREEBSD
 #   define ZYAN_POSIX
+#elif defined(__NetBSD__)
+#   define ZYAN_NETBSD
+#   define ZYAN_POSIX
```

Comment:
I think ZYAN_UNIX should be defined here as well, right? From what I've read, NetBSD seems to be an Unix based system.

Reply:
Well ZYAN_UNIX is not defined for Solaris or FreeBSD that are as much unix as NetBSD.
So that why NetBSD does not have it.

Reply:
Alright! We might have to revisit that at some point, but for now it should be fine.